modules/service/src/main/scala/CookieService.scala

```diff
@@ -95,7 +95,7 @@ object CookieWriter {
           name     = CookieName,
           domain   = Some(domain),
           content  = "",
-          sameSite = if (secure) SameSite.Strict else SameSite.Lax,
```

Comment:
`SameSite.Lax` actually requires https so it doesn't work locally. All very confusing. In any case this fixes the problem and you can run SSO locally with this change.